src/main/resources/reference.conf

```diff
@@ -201,10 +237,10 @@ cassandra-query-journal {
   
   # How many events to fetch in one query (replay) and keep buffered until they
   # are delivered downstreams.
-  max-buffer-size = 100
+  max-buffer-size = 10000
```

Comment:
I think that was the reason why I used 50 and 100 (2x). I don't mind increasing both those somewhat, but it is hard to say what is a good value. Gut feeling is that 10000 is too much. Perhaps 250 & 500?